services/QuillLMS/deploy.sh

```diff
@@ -117,6 +117,12 @@ then
         # Documented here: https://github.com/empirical-org/test_repo/blob/destination_branch/test_file.txt
         git fetch origin develop
         git fetch origin $DEPLOY_GIT_BRANCH
+
+        latest_prod_release_sha=`heroku releases -a empirical-grammar | grep Deploy | cut -d ' ' -f 4 | head -n 1`
```

Comment:
Note that when the previous deploy was a promotion, the sha of the promoted slug will differ from the sha of the merged PR (because of squash/rebase). Thus, `git log` will print the most recent commit as if it is not in production. I added the `--stat` flag to make this more obvious. Is this something we want?

Alternatives include:
- don't print anything / close this PR
- print the diff (we lose the commit messages this way)

Reply:
This feels a little confusing.  I would be fine with either alternative you list (print nothing or just print diff). Sorry for the trouble.